src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/submission/service/SubmissionServiceImpl.java

```diff
@@ -322,15 +324,17 @@ private long getExpiryDate() {
         return System.currentTimeMillis() + cacheProperties.getRedis().getTimeToLive().toMillis();
     }
 
-    private BigDecimal setDocumentId(ActionDocument actionDocument) {
+    private ActionDocument setActionDocument(ca.bc.gov.open.jag.efilingapi.api.model.ActionDocument actionDocument) {
 
         if (actionDocument == null) return null;
 
-        if (actionDocument.getType().equals(Keys.REJECTED_DOCUMENT_CODE)) return null;
+        logger.info("setting action document");
```

Comment:
this seems so underwhelming

Reply:
Yeah, it could be removed it is just a nice to know...